docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -35,7 +35,13 @@ Refer to the [YSQL tab](../point-in-time-restore-ysql) for details on this featu
 
 ## Try out the PITR feature
 
-You can test the PITR feature (BETA) by creating a database and populating it, creating a snapshot, and restoring ([data only](#limitations)!) from that snapshot.
+You can test the PITR feature (BETA) by creating a database and populating it, creating a snapshot, and restoring (be sure to check out the [limitations](#limitations)!) from that snapshot.
```

Comment:
not sure about the verbiage here, as in YSQL we call them databases, but in YCQL we call them namespaces

also, for the new API for PITR, we don't manually create snapshots, but rather create schedules that will automatically take snapshots

Reply:
I changed it to namespace just now, but isn't it actually keyspace?